src/platform/site-wide/announcements/components/ClaimIncreaseBanner.jsx

```diff
@@ -1,10 +1,7 @@
 import React from 'react';
 import backendServices from '../../../../platform/user/profile/constants/backendServices';
-import brandConsolidation from '../../../brand-consolidation';
 
 export default function ClaimIncreaseBanner({ dismiss, isLoggedIn, profile }) {
-  if (brandConsolidation.isEnabled()) return null;
```

Comment:
do we need this component at all now? looks like it did nothing on VA.gov

Reply:
We don't, but I was giving the benefit of a doubt that we may decide to show it again. Probably not though; I'll remove it.